syntax = "proto3";

package deployment;


// Represents a single host in the deployment
message Host {
  string hall = 1;
  string aisle = 2;
  int32 rack = 3;
  int32 shelf_u = 4;
  string host = 5;
  string pod_type = 6;
}

// Represents the overall deployment configuration
message DeploymentDescriptor {
  repeated Host hosts = 2;
}
